.github/workflows/unit-testing.yaml

```diff
@@ -1,6 +1,6 @@
 # Build the gtest Hector and run it
 
-name: googletest Hector
+name: Hector unit tests
```

Comment:
To clarify, these unit tests are different that the R hector test that unit tests correct?

Reply:
Looking through the rest of the PR this is pretty obvious... 🤦‍♀️